src/pdf/ShoppingList/ShoppingListPDF.tsx

```diff
@@ -0,0 +1,225 @@
+"use client";
+
+import { Document, Page, Text, View, StyleSheet } from "@react-pdf/renderer";
+import {
+    ClientSummary,
+    HouseholdSummary,
+    Item,
+    ParcelInfo,
+    RequirementSummary,
+    ShoppingListPDFProps,
+} from "@/pdf/ShoppingList/dataPreparation";
+import React from "react";
+
+const styles = StyleSheet.create({
+    paper: {
+        margin: "0.75in",
+        lineHeight: 1.5,
+    },
+    cell: {
+        display: "flex",
+        padding: "20 5 5",
+    },
+    infoBlock: {
+        display: "flex",
+        flexDirection: "row",
+        borderStyle: "solid",
+        border: 1,
+    },
+    infoCell: {
+        display: "flex",
+        width: "50%",
+        padding: 5,
+    },
+    table: {
+        display: "flex",
+        flexDirection: "column",
+        borderStyle: "solid",
+        border: 1,
+        padding: 5,
+    },
+    tableRow: {
+        display: "flex",
+        flexDirection: "row",
+        borderBottomStyle: "solid",
+        borderBottom: 1,
+        paddingVertical: 4,
+        textAlign: "center",
+    },
+    tableItemDescription: {
+        width: "25%",
+    },
+    tableQuantity: {
+        width: "35%",
+    },
+    tableNotes: {
+        width: "30%",
+    },
+    tableDone: {
+        width: "10%",
+    },
+    title: {
+        fontSize: "30pt",
+        fontFamily: "Helvetica-Bold",
```

Comment:
Is this font provided by every browser on every platform we're targeting? 🌍

Reply:
Yep, just checked that it is :)